src/Recommendations/Recommendations.js

```diff
@@ -12,13 +12,20 @@ const Recommendations = ({ animes, genre }) => {
 
   const getDetails = (url) => {
     getAnimes(url)
-      .then(data => setDetails(data))
+      .then(data => {
+        setDetails(data)
+        returnToTop();
+      })
       .catch(err => {
         setError(err);
-        console.log(error)
       })
   }
-  
+
+  const returnToTop = () => {
```

Comment:
This redirects a user to the top of the page when more the learn more button is clicked and the details card is displayed